metastore/scripts/upgrade/mysql/018-HIVE-6757.mysql.sql

```diff
@@ -15,7 +15,7 @@ WHERE
 ;
 
 UPDATE SERDES
-  SET SLIB='org.apache.hadoop.hive.ql.io.parquet.serde.ParquetHiveSerDe'
+  SET SLIB='org.apache.hadoop.hive.ql.io.parquet.serdes.ParquetHiveSerDe'
```

Comment:
I think you did some kind of global search & replace on serde->serdes that changed a bunch of places like this that should not have been changed.